src/ExpressServer.ts

```diff
@@ -0,0 +1,44 @@
+import express from "express";
+import { Environment } from "./utils/constants";
+import { SetHueTokens } from "./utils/helpers";
+
+export default class ExpressServer {
+    private _server;
+
+    constructor() {
+        this._server = express();
+
+        this._server.get("/", (request, response) => {
+            response.send("Discord bot active.");
+        });
+        
+        // hue auth flow configuration
+        this._server.get("/seabot_hue", async (request, response) => {
+            try {
+                const { code, state } = request?.query;
+                if (!state || state != Environment.hueState) {
+                    throw new Error("Invalid state value");
```

Comment:
Should we output the actual failure state?

Reply:
This is old code that we can likely ignore / remove eventually - but this is just a very very insecure security check to ensure the request to link the Hue API to my bridge occurred from this app - as the state value is stored in the web app config